phoonnx/thirdparty/bw2ipa.py

```
@@ -1,66 +1,207 @@
+        return prev_ipa + "ː"
+
+    if prev_token in CONSONANTS:
+        # consonant gemination: use length mark, not duplication
+        if prev_ipa.endswith("ː"):
+            return prev_ipa  # already geminated
+        return prev_ipa + "ː"
+
+    return prev_ipa
+
+
+def mantoq_to_ipa(text, add_stress=True):
```

Ruff (0.14.10)

Unused function argument: `add_stress`

(ARG001)